snippets/liquid-template-snippets.json

```diff
@@ -589,6 +716,672 @@
+  "Month end dates drop": {
+    "prefix": ".month_end_dates",
+    "body": ".month_end_dates",
+    "description": "Return an array of all month end dates in the fiscal year - must use with variable of type period (array)"
+  },
+  "People drop": {
+    "prefix": ".people",
+    "body": ".people",
+    "description": "Return all people in the period - must use with variable of type period (drop)"
+  },
+  "Reconciliations drop": {
+    "prefix": ".reconciliations",
+    "body": ".reconciliations",
+    "description": "Return all reconciliations in the period - must use with variable of type period (drop)"
+  },
+  "Reports drop": {
+    "prefix": ".reports",
+    "body": ".reports",
+    "description": "Return collection of Reports for the period - must use with variable of type period (drop)"
+  },
+  "Report drop": {
+    "prefix": ".reports.[report handle]",
+    "body": ".reports.${1:[report handle]}",
+    "description": "Return specific report for the period - must use with variable of type period (drop)"
+  },
+  "Shareholders drop": {
+    "prefix": ".shareholders",
+    "body": ".shareholders",
+    "description": "Return all shareholders in the period - must use with variable of type period (special drop)"
+  },
+  "Start date drop": {
+    "prefix": ".start_date",
+    "body": ".start_date",
+    "description": "Return start date of the period - must use with variable of type period (date)"
+  },
+  "Year end drop": {
+    "prefix": ".year_end",
+    "body": ".year_end",
+    "description": "Return period drop of period at end of fiscal year - must use with variable of type period (drop)"
+  },
+  "Year end date drop": {
+    "prefix": ".year_end_date",
+    "body": ".year_end_date",
+    "description": "Return end date of the fiscal year - must use with variable of type period (date)"
+  },
+  "Year start date drop": {
+    "prefix": ".year_start_date",
+    "body": ".year_start_date",
+    "description": "Return start date of the fiscal year - must use with variable of type period (date)"
+  },
+  "Minus period/year": {
+    "prefix": ".minus_",
+    "body": ".minus_${1|1,2,3|}${2|p,y|}",
+    "description": "Return period before current period by a set number of periods or year - must use with variable of type period (drop)"
+  },
+  "Plus period/year": {
+    "prefix": ".plus_",
+    "body": ".plus_${1|1,2,3|}${2|p,y|}",
+    "description": "Return period after current period by a set number of periods or year - must use with variable of type period (drop)"
+  },
+
+  "Reconciliations drop (with options)": {
+    "prefix": ".reconciliations.",
+    "body": ".reconciliations.${1|[reconciliation_handle],count,star|}",
+    "description": "All drops under the Reconciliations drop"
+  },
+  "Reconciliation star drop": {
+    "prefix": ".star",
+    "body": ".star",
+    "description": "Return all starred reconciliations - must use with variable of type reconciliations (drop)"
+  },
+  "Reconciliation star drop (with options)": {
```

Comment:
This should be `starred?` ?

Reply:
Seems to just be 'starred' per guidance? (https://developer.silverfin.com/docs/reconciliations), but have otherwise updated

Reply:
You're right, I was looking at the ReconciliationDrop (https://developer.silverfin.com/docs/reconciliation) instead of the ReconciliationsDrop